public_html/wp-content/plugins/wc-post-types/inc/rest-api.php

```diff
@@ -147,8 +155,60 @@ function register_additional_rest_fields() {
 		]
 	);
 }
+
 add_action( 'rest_api_init', __NAMESPACE__ . '\register_additional_rest_fields' );
 
+/**
+ * Validate simple meta query parameters in an API request and add them to the args passed to WP_Query.
+ *
+ * @param array $args    The prepared args for the WP_Query object.
+ * @param array $request The args from the REST API request.
+ *
+ * @return array
+ */
+function prepare_meta_query_args( $args, $request ) {
+	if ( isset( $request['meta_key'], $request['meta_value'] ) ) {
+		$args['meta_key']   = $request['meta_key'];
+		$args['meta_value'] = $request['meta_value'];
+	}
+
+	return $args;
+}
+
+add_filter( 'rest_wcb_session_query', __NAMESPACE__ . '\prepare_meta_query_args', 10, 2 );
+
+/**
+ * Add meta field schemas to Sessions collection parameters.
+ *
+ * This enables and validates simple meta query parameters for the Sessions endpoint. Specific meta keys are
+ * safelisted by filtering for ones that have `show_in_rest` set to `true`.
+ *
+ * @param array        $query_params
+ * @param WP_Post_Type $post_type
+ *
+ * @return array
+ */
+function add_meta_collection_params( $query_params, $post_type ) {
+	$public_meta_fields = array_filter( get_registered_meta_keys( 'post', $post_type->name ), function( $registered ) {
```

Comment:
```suggestion
	// Avoid exposing potentially sensitive data.
	$public_meta_fields = array_filter( get_registered_meta_keys( 'post', $post_type->name ), function( $registered ) {
```

`SHOULD` - I think the security issues are implied by the docblock comment and the name of this var, but it's worth being extra safe and declaring it explicitly.